data/cisagov_R.yml

```diff
@@ -34,4 +34,596 @@ software:
     references:
       - ''
     last_updated: '2022-01-12T07:18:55+00:00'
+  - vendor: Red Hat
+    product: Red Hat Enterprise Linux 6 Extended Lifecycle Support
+    cves:
+      cve-2021-4104:
+        investigated: true
+        affected_versions:
+          - Red Hat Enterprise Linux 6 Extended Lifecycle Support
```

Comment:
If it is/was affected, but now fixed, just put the version under fixed and leave affected_versions: []